sippy-ng/src/releases/ReleasePayloadTable.js

```diff
@@ -58,7 +68,24 @@ function ReleasePayloadTable(props) {
         }
       },
     },
-
+    {
+      field: 'forced',
+      headerName: 'Forced',
+      align: 'center',
+      flex: 0.75,
+      hide: props.briefTable,
+      renderCell: (params) => {
+        if (params.value === true) {
+          return (
+            <Tooltip title="This payload was forced.">
```

Comment:
```suggestion
            <Tooltip title="This payload was manually force accepted/rejected.">
```

Is this an improvement?